src/components/Person/Person.jsx

```diff
@@ -1 +1,34 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import './Person.scss';
```

Comment:
Try to fix this error
![image](https://user-images.githubusercontent.com/70009205/215254953-0da0b20b-7160-48eb-84ab-849145c2906a.png)
If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review